test/TestSuite.hx

```diff
@@ -26,20 +26,20 @@ class TestSuite extends massive.munit.TestSuite
 	{
 		super();
 
-		add(massive.munit.AssertionExceptionTest);
 		add(massive.munit.AssertTest);
-		add(massive.munit.async.AsyncDelegateTest);
+		add(massive.munit.TestSuiteTest);
```

Comment:
revert this block, its not supposed to be changed

Reply:
I didn't manually changed it it's after I run the build.sh